tests/providers/sftp/sensors/test_sftp.py

```diff
@@ -86,16 +86,15 @@ def test_file_new_enough(self, sftp_hook_mock):
 
     @patch("airflow.providers.sftp.sensors.sftp.SFTPHook")
     def test_file_not_new_enough(self, sftp_hook_mock):
```

Comment:
It would male sense to update for the test_file_new_enough test case as well.

Reply:
Yes, once the solution to https://github.com/apache/airflow/issues/36629 is found can update.